fuzzypp/src/preprocessor.hpp

```diff
@@ -0,0 +1,24 @@
+#ifndef FUZZYPP_PREPROCESSOR_HPP
+#define FUZZYPP_PREPROCESSOR_HPP
+
+#include <string>
+
+#include <simplecpp.h>
+
+#include "cliopts.hpp"
+
+namespace fuzzypp::preprocessor {
+    class FuzzyPreprocessor {
```

Comment:
Silly note, but the preprocessor itself isn't fuzzy ;). This name can be misleading, perhaps just `Preprocessor` or `Fuzzyc2cpgPreprocessor`, or `Fuzzyc2cpgPP`?